autogluon/utils/tabular/features/abstract_feature_generator.py

```diff
@@ -102,7 +102,8 @@ def fit_transform(self, X: DataFrame, y=None, banned_features=None, fix_categori
             if unique_value_count == 1:
                 self.features_to_remove_post.append(column)
             # TODO: Consider making 0.99 a parameter to FeatureGenerator
-            elif 'object' in self.feature_type_family and column in self.feature_type_family['object'] and (unique_value_count / X_len > 0.99):
+            elif ('object' in self.feature_type_family and column in self.feature_type_family['object'] and (unique_value_count / X_len > 0.99) 
+                and column not in self.feature_type_family['text']):
```

Comment:
add `('text' not in self.feature_type_family or column not in self.feature_type_family['text'])` , to avoid case where 'text' doesn't exist in feature_type_family being accessed.